MapboxNavigation/RouteMapViewController.swift

```diff
@@ -99,34 +100,39 @@ class RouteMapViewController: UIViewController {
      */
     var annotatesSpokenInstructions = false
 
+    override func loadView() {
+        let nav = NavigationView(delegate: self)
+        self.view = nav
+        self.navView = nav
+    }
     override func viewDidLoad() {
         super.viewDidLoad()
         automaticallyAdjustsScrollViewInsets = false
         
         distanceFormatter.numberFormatter.locale = .nationalizedCurrent
         
-        mapView.tracksUserCourse = true
-        mapView.delegate = self
-        mapView.navigationMapDelegate = self
-        mapView.courseTrackingDelegate = self
-        mapView.contentInset = contentInsets
-        
-        rerouteReportButton.slideUp(constraint: rerouteFeedbackTopConstraint)
-        rerouteReportButton.applyDefaultCornerRadiusShadow(cornerRadius: 4)
-        overviewButton.applyDefaultCornerRadiusShadow(cornerRadius: overviewButton.bounds.midX)
-        reportButton.applyDefaultCornerRadiusShadow(cornerRadius: reportButton.bounds.midX)
-        muteButton.applyDefaultCornerRadiusShadow(cornerRadius: muteButton.bounds.midX)
-        
-        wayNameLabel.clipsToBounds = true
-        wayNameLabel.layer.borderWidth = 1.0 / UIScreen.main.scale
-        wayNameLabel.applyDefaultCornerRadiusShadow()
-        lanesView.isHidden = true
-        statusView.isHidden = true
-        statusView.delegate = self
-        nextBannerView.isHidden = true
-        isInOverviewMode = false
-        instructionsBannerView.delegate = self
-        bottomBannerView.delegate = self
+//        mapView.tracksUserCourse = true
+//        mapView.delegate = self
+//        mapView.navigationMapDelegate = self
```

Comment:
There are more delegates on the `NavigationMapView` than just `.delegate`, fix this.